include/RAJA/pattern/WorkGroup/SimpleVector.hpp

```diff
@@ -0,0 +1,173 @@
+/*!
+ ******************************************************************************
+ *
+ * \file
+ *
+ * \brief   Header file providing RAJA SimpleVector for workgroup.
+ *
+ ******************************************************************************
+ */
+
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef RAJA_PATTERN_WORKGROUP_SimpleVector_HPP
+#define RAJA_PATTERN_WORKGROUP_SimpleVector_HPP
+
+#include "RAJA/config.hpp"
+
+#include <utility>
+
+
+namespace RAJA
+{
+
+namespace detail
+{
+
+/*!
+ * A simple vector like class
+ */
+template < typename T, typename Allocator >
+struct SimpleVector
```

Comment:
Can this be merged with ``RAJAVec.hpp`` . I would also be in favor of a more descriptive name.

Reply:
I missed RAJAVec, that's a good idea.

Reply:
Combined.